measurement-kit-test/NetworkMeasurement.h

```diff
@@ -10,7 +10,6 @@
 @property NSNumber *test_id;
 @property NSString *status;
 @property BOOL finished;
-@property NSMutableArray *logLines;
```

Comment:
I removed this because we can pass the logLines directly in the dispatched message.